Sources/Network/JSON.swift

```diff
@@ -361,7 +361,7 @@ public enum JSON {
     ///   - parseAPIError: The API error parsing closure.
     /// - Returns: The value of type `T` associated to the given attribute's key.
     /// - Throws: An error of type `JSON.Error`, or a domain specific `Swift.Error` produced by `parseAPIError`.
-    private static func parseValue<T>(rawValue: Any,
+    internal static func parseValue<T>(rawValue: Any,
```

Comment:
We can perhaps simply omit the modifier, or do you think it's worth it?
Additionally, please align the parameters correctly 👮 💄

Reply:
ups 😇 and yes we can. i thought xcode would recommend to remove the modifier, but it didn't 🙄